### ISSUE TITLE
Upstream changed signature in v4.5.2

### ISSUE THREAD

**Issue by wizhippo** [netgen/ibexa-site-api#22]
https://github.com/netgen/ibexa-site-api/blob/928abfb3501717efb88889bf2b8b61918be97e2e/bundle/Core/FieldType/RichText/Renderer.php#L18

https://github.com/ibexa/fieldtype-richtext/commit/7468d4961eaff40263a6a411ac1fc6df5147d545

**Comment by pspanja** [comment]
Thanks @wizhippo, we have it on radar and willl fix it soonish.

**Comment by dfritschy** [comment]
Unfortunately this also affects the installation of https://github.com/netgen/media-site - installation is fine, but as soon as a RichText field is rendered, this error is produced:
```
Ibexa\FieldTypeRichText\RichText\Renderer::__construct(): Argument #2 ($configResolver) must be of type Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface, Symfony\Component\Security\Core\Authorization\AuthorizationChecker given, called in /Users/dfritschy/daten/htdocs/media-site-il2023/vendor/netgen/ibexa-site-api/bundle/Core/FieldType/RichText/Renderer.php on line 31
```

**Comment by ilukac** [comment]
@dfritschy unfortunately Petar is on vacation this month, not sure when he will have time to check this :/

**Comment by dfritschy** [comment]
@ilukac I'm aware that this is the result of a breaking change Ibexa introduced without proper versioning. However this renders a fresh installation or update of Netgen Media site with current `composer.json` version restrictions unusable.

**Comment by ilukac** [comment]
@emodric is this fixable only with composer.json fix maybe?

**Comment by emodric** [comment]
No, needs update in code. Working on it now.

**Comment by emodric** [comment]
Fixed in 5.3.0. Feel free to reopen this if more issues arise @wizhippo @dfritschy !

**Comment by dfritschy** [comment]
@ilukac @emodric Looks good at first sight. Thanks for the fix!
